pkg/controller/kogitoinfra/infinispan/resource.go

```diff
@@ -23,17 +25,17 @@ import (
 	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/logger"
 	"k8s.io/apimachinery/pkg/api/errors"
-	"reflect"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	// InstanceName is the default name for the Infinispan provisioned instance
-	InstanceName     = "kogito-infinispan"
-	secretName       = "kogito-infinispan-credential"
-	identityFileName = "identities.yaml"
+	InstanceName = "kogito-infinispan"
+	secretName   = "kogito-infinispan-credential"
+	// IdentityFileName is the name of YAML file containing list of Infinispan credentials
+	IdentityFileName = "identities.yaml"
```

Comment:
Formatting of constants is a little bit off.

Reply:
The formatting changed because I have added there a comment.
Comment was added to satisfy go-lint as I changed the IdentityFileName to be exported.

Reply:
I see.